scrooge-generator/src/main/resources/javagen/finagleServiceFunction.java

```diff
@@ -3,17 +3,18 @@
     try {
```

Comment:
what is returned at line 8 is a CompletableFuture because of the interface change. Therefore I have to change the result type.  I will show you an example of a generated file tomorrow and that should make things clear.